### DISCUSSION TITLE
Is next/image "onLoadingComplete" really working?

### DISCUSSION THREAD

**Discussion by vitorregisrr** [vercel/next.js#27152] [Help] [answered]
Hey there!

In [next/image docs ](https://nextjs.org/docs/api-reference/next/image) we can see a property called "onLoadingComplete"... But It isn't working here (not throwing the callback) and my editor is saying that there isn't this prop in the types.

May can you help? Ty!

![image](https://user-images.githubusercontent.com/9900252/125532160-23d74398-ab10-45ba-be2e-52b6f207df42.png)
![image](https://user-images.githubusercontent.com/9900252/125532318-c80d2d59-ed4d-4f2a-ac7c-c73cf827d310.png)
(my editor is in Portuguese)

**Comment by nickadamson** [comment, marked as answer]
Not working for me either. 

Opened issue #27213

**Reply by martinbavio** [reply]
Same for me:

<img width="1080" alt="Screen Shot 2021-07-22 at 6 50 26 PM" src="https://user-images.githubusercontent.com/5664/126714722-22081541-60ce-4a1d-b384-f02982669b18.png">

**Reply by leerob** [reply]
Given this a bug, I'm going to lock this discussion and move the conversation to the Issue. Thank you! 

If you could add a reproduction to the Issue, that would be very helpful 🙏

**Reply by styfle** [reply]
This prop is not available on stable yet, you'll need to use the canary channel.

You can try it out today with `yarn add next@canary`

https://github.com/vercel/next.js/issues/18398#issuecomment-872604823
